script_library/deployment-actions-openstack.sh

```diff
@@ -81,6 +81,10 @@ function clean_airship(){
     fi
 }
 function teardown(){
+    if [[ ${SOCOK8S_GATHER_LOGS:-"NO"} == "YES" ]]
```

Comment:
if we are exposing a new feature, I don't see why we should hide this in teardown.

Reply:
expose but not force the user to use it, let them choose if they want to ignore gathering logs or not. I dont see a problem here, after all gather_logs takes time and you might not be interested in spending any

Reply:
I see. It's just to avoid time.
I would still prefer being explicit than implicit. If you want to gather logs, just run the command before teardown, else don't run the command. It reduces the code we have to maintain, and code paths we have to remember.